TileDB.Cloud/RestUtil.cs

```diff
@@ -455,10 +328,91 @@ public static (string name_space, string array_path) SplitUri(string array_uri)
             System.Uri uri = new Uri(array_uri);
             if (uri.Scheme != "tiledb")
             {
-                throw new System.ArgumentException("Invalid array uri(not starting with tiledb):" + array_uri);
+                throw new ArgumentException("Invalid array uri(not starting with tiledb):" + array_uri);
             }
             return (uri.Host, uri.LocalPath.TrimStart('/'));
         }
 
+        public static string SerializeArguments(Dictionary<string, dynamic> args)
+        {
+            return JsonConvert.SerializeObject(args);
+        }
```

Comment:
I don't think this method should be public; it is too abstract. Can you inline it in the example?